test/data/testinit.js

```diff
@@ -32,14 +32,14 @@ this.q = function() {
 
 /**
  * Asserts that a select matches the given IDs
- * @param {String} a - Assertion name
- * @param {String} b - Sizzle selector
- * @param {String} c - Array of ids to construct what is expected
- * @example t("Check for something", "//[a]", ["foo", "bar"]);
- * @result returns true if "//[a]" return two elements with the IDs 'foo' and 'bar'
+ * @param {String} message - Assertion name
+ * @param {String} selector - Sizzle selector
+ * @param {String} expectedIds - Array of ids to construct what is expected
+ * @param {String} context - Expected `undefined` or `#qunit-fixture`
```

Comment:
The type should be `{(String|Node)=document}` (cf. [`jQuery`](http://api.jquery.com/jQuery/)), and the description should merely document it as "selection context".